presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestJtsGeometryUtils.java

```diff
@@ -93,15 +93,19 @@ private static void assertGeometry(String wkt)
         Slice geometry = stGeometryFromText(utf8Slice(wkt));
         Geometry expected = WKT_READER.read(wkt);
         Geometry actual = JtsGeometryUtils.deserialize(geometry);
+        Slice serialized = JtsGeometryUtils.serialize(expected);
```

Comment:
with this addition the flow of this method is confusing to me;

how about moving extra testing to the end of this method?

```
if (expected.isEmpty()) {
   assertTrue(deserialize(JtsGeometryUtils.serialize(expected)).isEmpty());
}
else {
   assertEquals(deserialize(JtsGeometryUtils.serialize(expected)), expected);
}
```